src/jquery-sticky-header-footer.js

```diff
@@ -60,8 +61,17 @@
                 /**
                     Remove added DOM elements and plugin data
                  */
-                $('.' + classNames.outerWrapper).before(element).remove();
-                element.removeData('plugin_' + pluginName);
+                $.each(['footerElement', 'headerElement'], function(idx, val) {
```

Comment:
I think either would work, but your way is cleaner. I meant to go back and change it but forgot.